api/v1/service.proto

```
@@ -0,0 +1,76 @@
+service QueryService {
+  rpc Query(QueryRequest) returns (QueryResponse) {}
+}
+
+service CacheService {
+  rpc Cache(google.protobuf.Empty) returns (google.protobuf.Empty) {}
+  rpc Clear(google.protobuf.Empty) returns (google.protobuf.Empty) {}
+}
+
+service LeaderboardService {
+  rpc CustomLeaderboard(CustomLeaderboardRequest) returns (CustomLeaderboardResponse) {}
+  rpc AllKeys(google.protobuf.Empty) returns (AllKeysResponse) {}
```

buf

"google.protobuf.Empty" is used as the request or response type for multiple RPCs.

(RPC_REQUEST_RESPONSE_UNIQUE)